data-raw/woke_windows.R

```diff
@@ -30,12 +30,16 @@ boston_pd_1120 <- boston_pd_1120 |>
       race == "asian" ~ "ASIAN",
       race == "UNK" ~ "UNKNWN",
       TRUE ~ race)
-  )
+  ) |>
+  rename(disposition_desc = disposition_desc_15) |>
+  select(-c(disposition_desc_40, race)) |>
+  sample_n(150000)
```

Comment:
I'm thinking now about whether a random sample is preferable to a set of years...

Reply:
As per the comment above, we could add something like:

`mutate(event_timestamp = lubridate::parse_date_time(...))`